### ISSUE TITLE
JS error in non-AMP posts when there's no sponsor info to display

### ISSUE THREAD

**Issue by dkoo** [Automattic/newspack-theme#1030]
**Describe the bug**
When viewing a non-AMP post that has no sponsor info to display, a JS error will be thrown:

```
Uncaught TypeError: Cannot read property 'parentNode' of null
    at eval (amp-fallback-newspack-sponsors.js:9)
```

This is probably because [this `getElementById` call](https://github.com/Automattic/newspack-theme/blob/master/newspack-theme/js/src/amp-fallback-newspack-sponsors.js#L9) returns `null`, so the subsequent element methods fail:

**To Reproduce**
1. For any post, make sure there are no sponsors applied to the post, either as a direct sponsor or to its categories or tags.
2. Disable AMP site-wide, or view a post with AMP disabled for the post.
3. View the post and observe the JS error in the console.

**Screenshots**
n/a

**Expected behavior**
Probably just need to return if the `getElementById` call returns something falsy.

**Comment by dkoo** [comment]
Sorry I missed this during PR testing!

**Comment by laurelfulford** [comment]
Oh, good find!

> Sorry I missed this during PR testing!

I'm sorry I introduced it in the first place 😄

**Comment by dkoo** [comment]
Related PHP error:

```
[17-Aug-2020 21:58:39 UTC] PHP Warning:  Invalid argument supplied for foreach() in /Users/dkoo/Local Sites/newspack/app/public/wp-content/themes/repo/newspack-theme/inc/newspack-sponsors.php on line 62
```

If `$sponsors_all` is `false` we should return early here: https://github.com/Automattic/newspack-theme/blob/master/newspack-theme/inc/newspack-sponsors.php#L58

**Comment by laurelfulford** [comment]
Thanks Derrick! I ran into that when testing a fix for this issue, too. Should be fixed with #1032!
